layers/deft/README.org

```diff
@@ -16,7 +16,12 @@ browsing and creating notes.
 ** Differences from default
 
 This layer sets the default to use filenames for note titles as well as
-=org-mode= for editing. The default extension is still =.txt= though.
+=org-mode= for editing. The default extension is =org=. Configure by adding the
+following in your config file: 
+
+#+Begin_SRC emacs-lisp
+  (setq deft-extensions '("your" "extension" "here"))  
+#+END_SRC
 
```

Comment:
Change to: 
+#+Begin_SRC emacs-lisp
+  (setq deft-extensions '("org" "md" "txt"))  
+#+END_SRC